src/writer/to_code.rs

```diff
@@ -116,12 +116,23 @@ impl ToCode for Chunk {
                 let s = format_block_one_line(&prefix, suffix, &value_strings, "", "");
                 vec![s]
             }
-            ErrorResultReturn { ref value } => {
+            AssertErrorSanity => {
+                let condition = r#"(is_ok == 0) ^ error.is_null()"#;
```

Comment:
Wut. O.o

Reply:
What's wrong with that line? :) You don't like the `r#` or the xor?

Reply:
The xor.

Reply:
How would you express that more readable? Either one or the other must be true, but never both at once or none of the two.

Reply:
Something like this:

```
(is_ok == 0) != error.is_null()
```

?

Reply:
Yes there are certainly many other ways of writing that same condition.
If you prefer an equivalence check, perhaps you'd like an `assert_eq!()` instead so that it will also show the values on failures?

Reply:
As long as it's easier to read, it's fine.